Calibration/HcalCalibAlgos/src/RecAnalyzerMinbias.cc

```diff
@@ -35,58 +36,64 @@ class RecAnalyzerMinbias : public edm::EDAnalyzer {
 public:
   explicit RecAnalyzerMinbias(const edm::ParameterSet&);
   ~RecAnalyzerMinbias();
-  virtual void analyze(const edm::Event&, const edm::EventSetup&);
-  virtual void beginJob() ;
-  virtual void endJob() ;
+
+  virtual void analyze(const edm::Event&, const edm::EventSetup&) override;
```

Comment:
This can definitely not be global since it is filling a TFile and one can't work with the same TFile on multiple threads.